APCAppCore/APCAppCore/UI/Onboarding/APCStudyOverviewCollectionViewController.m

```diff
@@ -77,6 +77,12 @@ - (void)viewDidLoad {
     self.items = [self prepareContent];
     [self setUpPageView];
     [self setupCollectionView];
+        
+    if (self.transitionToSignInOnLoad)
+    {
+        self.transitionToSignInOnLoad = NO;
+        [self signInTapped:self];
```

Comment:
So... this won't work for mPower. The name suggests that this is a transition, i.e.. animated. If you look at signInTapped it can either be a push onto a navigation stack -or- presentViewController. Either way it is animated. Trying to animate a presentation off of a view controller that has not yet been displayed can result in strange things.